src/components/ActivityPage/Transaction/Transaction.tsx

```diff
@@ -373,24 +373,6 @@ const Transaction = (props: Props) => {
       )
     }
 
-    case PUBLISH_THIRD_PARTY_ITEMS_SUCCESS: {
-      const { collectionId, collectionName, items } = tx.payload
-      return (
-        <TransactionDetail
-          tx={tx}
-          collectionId={collectionId}
-          text={
-            <T
-              id="transaction.publish_third_party_items"
```

Comment:
we'd have to delete the translations too right?

Reply:
yes! removed, thanks! :)